File: 3242-CountElementsWithMaximumFrequency/3242-CountElementsWithMaximumFrequency.py
# Last updated: 31/07/2025, 21:05:59
class Solution:
    def maxFrequencyElements(self, nums: List[int]) -> int:
        arr = []
        c = 0
        s = 0
        n=0
        while nums:
            i = nums[0]
            n = nums.count(i)
            arr.append(n)
            for _ in range(n):
                nums.remove(i)
        c = max(arr)
        s = c * arr.count(c)
        return s

File: 3242-CountElementsWithMaximumFrequency/3242-CountElementsWithMaximumFrequency.java
// Last updated: 31/07/2025, 21:05:52
class Solution {
    public int maxFrequencyElements(int[] nums) {
        int[] freq=new int[101];
        for(int i=0;i<nums.length;i++)
        {
            freq[nums[i]]++;
        }
        int max=0;
        for(int f:freq)
        {
            max=Math.max(f,max);
        }
        int c=0;
        for(int f:freq)
        {
            if(f==max)
            {
                c++;
            }
        }
        return c*max;
    }
}